_includes/post_preview.html

```diff
@@ -0,0 +1,24 @@
+<span class="post-meta">
+  {{ post.date | date: "%b %-d, %Y" }}
+</span>
+
+<h2>
+  <a class="post-link" href="{{ post.url | prepend: site.baseurl }}">{{ post.title }}</a>
+</h2>
+
+<div class="post-content" itemprop="articleBody">
+  {{ post.content | markdownify | strip_html | truncatewords: site.post_preview_words }}
```

Comment:
Is there any way to truncate/strip without stripping everything? E.g. having paragraphs to break up the first couple sections of text may be helpful to scan an article (especially in combination with an expanded value for `post_preview_words`).

Reply:
Not completely sure, but I bet we can do better than what I have here. I'll spend a little time digging into this and see what I can figure out.

Reply:
Found a simple plugin that does the job pretty well

![image](https://user-images.githubusercontent.com/12214617/97522108-99a3b980-1964-11eb-9cfd-bebbd892ffeb.png)